### PR TITLE
Use clean database url

### PR DESCRIPTION
**Pull request by jalet**
There's no need to parse the DATABASE_URL env variable, we can simply pass it as is.

Also updated a few queries to remove the deprecation warnings.

### CODE REVIEW

**Comment by jalet** [comment]
I just deployed this to staging, and it seams to work just fine.

File: src/server/database.js [hunk 1]
```diff
@@ -1,30 +1,17 @@
...
-)
+const Op = Sequelize.Op
+
+const sequelize = new Sequelize(CONFIG.DATABASE_URL, {
+  dialect: 'postgres',
+  protocol: 'postgres',
+  logging: process.env.NODE_ENV === "production" ? null : console.log,
+  operatorsAliases: false,
+  dialectOptions: {
+      ssl: process.env.NODE_ENV === "production" ? true : false,
```
Comment (dnjstrom) [inline src/server/database.js]: Maybe just use the boolean statement?

Comment (dnjstrom) [reply src/server/database.js]: `ssl: process.env.NODE_ENV === "production"`

Comment (jalet) [reply src/server/database.js]: like `Boolean(NODE_ENV==="production")?

Comment (dnjstrom) [reply src/server/database.js]: Why cast to Boolean, it should always be one, right?

Comment (jalet) [reply src/server/database.js]: haha, yeah ofc 💃

**Pull request merged**